pkg/obfuscate/sql_tokenizer_test.go

```diff
@@ -30,3 +32,66 @@ func TestSQLTokenizerPosition(t *testing.T) {
 	}
 	assert.Equal(10, tokenCount)
 }
+
+func TestTokenizeDecimalWithoutIntegerPart(t *testing.T) {
```

Comment:
Any reason to not have this also be a fuzz test? It will run by default with `go test` in the same way as it would as a unit test, it just has the added benefit that it can also be run with `go test -fuzz`

Reply:
I thought the other two fuzzing tests sufficiently covered the vast majority of cases, this was more of a special case that I thought I would quickly write tests for. It could definitely be changed into a fuzz test if desired.

Reply:
The generation of the fuzzed numbers would also be slightly more complex - the test was written because `FormatFloat` didn't seem to allow numbers to be formatted this way by default. The workaround would be randomly prepending either `-.` or `.` to fuzzily generated integers, as well as generating a random amount of leading 0's to the integer - which would be more complex than the other tests.